src/asserts/possible-phone-assert.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+import { PhoneNumberUtil } from 'google-libphonenumber';
+import { Violation } from 'validator.js';
+import { find } from 'lodash';
+
+/**
+ * Phone util instance.
+ */
+
+const phoneUtil = PhoneNumberUtil.getInstance();
+
+/**
+ * Export `PossiblePhoneAssert`.
+ */
+
+export default function possiblePhoneAssert(code) {
+  /**
+   * Optional peer dependencies.
+   */
+
+  const countries = require('isoc');
+
+  /**
+   * Class name.
+   */
+
+  this.__class__ = 'PossiblePhone';
+
+  // Test by `alpha-3` code.
+  let country = find(countries, { alpha3: code });
```

Comment:
Still debating whether we should accept _alpha3_ as well as _alpha2_. Maybe we should just accept _alpha2_ for simplicity's sake?

WDYT @ruimarinho?